src/Model/TransportModel/tsp1mvt1.f90

```diff
@@ -58,24 +59,21 @@ module GwtMvtModule
     procedure :: set_fmi_pr_rc
     procedure, private :: mvt_setup_outputtab
     procedure, private :: mvt_print_outputtab
-  end type GwtMvtType
+  end type TspMvtType
 
 contains
 
-  subroutine mvt_cr(mvt, name_model, inunit, iout, fmi1, gwfmodelname1, &
-                    gwfmodelname2, fmi2)
-! ******************************************************************************
-! mvt_cr -- Create a new initial conditions object
-! ******************************************************************************
-!
-!    SPECIFICATIONS:
-! ------------------------------------------------------------------------------
+  !> @brief Create a new mover transport object
+  !<
+  subroutine mvt_cr(mvt, name_model, inunit, iout, fmi1, eqnsclfac, & ! kluge note: does this need tsplab?
```

Comment:
would be good to figure out if tsplab is needed and remove the kluge note

Reply:
Great catch, @langevin-usgs. The tsplab class has been removed, this comment should have come out with it. Fixed with 99049d1